templates/LDA_Preprocessing.ipynb

```diff
@@ -3,7 +3,6 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    # or use manually specified cluster names
</pre><p>Will you make sure to run all of the notebooks to confirm everything is working? The spatial_LDA ones obviously, but also kmeans and enrichment analysis if those were affected at all. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>